electrum-env

```diff
@@ -34,5 +34,6 @@ export PYTHONPATH="$PYTHONPATH:"\
 "/usr/lib/python3/dist-packages:"\
 "/usr/lib/python${PYTHON_VER}/site-packages:"
 
+export PYTHONPATH="$PYHTONPATH:./env/lib/python${PYTHON_VER}/site-packages:./env/lib/python${PYTHON_VER}/dist-packages" 
```

Comment:
the only reason this "works" in your use case is because of the typo :P
`PYHTONPATH` (y<->h)